ansible/roles/linux-perf/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+#
+# installs Linux perf @ /usr/local/bin/perf
+#
+
+- stat: path=/usr/local/bin/perf
+  register: linux_perf
+  tags: linux_perf
```

Comment:
✂️ You can remove all of the tags

Reply:
Keep the tags. It allows for directed application:
```
ansible-playbook test*gaga --tags "linux_perf"
```

Reply:
if we want to keep the tags we should at least use a block and tag the block instead of adding tag on each task:

```
- name: install perf on linux
  block:
     - task 1
     - task 2
  tags: linux_perf
```

Reply:
Or leave the `stat` out of the block and also add the `when: not (linux_perf.stat.exists | bool)` on the block

```
- stat:
  tags: linux_perf
  register: linux_perf

- name: install perf on linux
  block:
     - task 1
     - task 2
  tags: linux_perf
  when: not (linux_perf.stat.exists | bool)
```